csrc/executor_kernel_arg.h

```diff
@@ -395,11 +419,9 @@ class TORCH_CUDA_CU_API KernelArgumentHolder {
  private:
   std::vector<std::unique_ptr<ArgAbstract>> arguments_;
   std::vector<void*> void_ptrs_;
-  bool changed_ = true;
```

Comment:
This was used for avoiding reallocating a buffer vector if nothing changed. Reallocation can be easily avoided by using `std::vector::resize`. The actual saving then is to avoid copying pointers, which should be negligible.